### ISSUE TITLE
Per-World specific permissions occasionally fail to work

### ISSUE THREAD

**Issue by Darktilldawn** [lucko/LuckPerms#226]
Permission listed below are some examples of nodes that are refusing to work when specified to a specific world on a specific server. When set to a specific server, and a global for world. They will work.
*agp.command.addleader*
*sponge.command.tps*
*nucleus.home.set.base*
*minecraft.commandblock* 

This is the /nucleus info for all the versions to be checked for up to date.
https://hastebin.com/silekakoro.sql

**Comment by lucko** [comment]
Only just had a chance to test this. Seems to work for me fine.

![](http://i.imgur.com/1czumIx.png)

In the config:
```hocon
# The name of the server, used for server specific permissions. Set to 'global' to disable.
server="test"

# If users on this server should have their global permissions applied.
# If set to false, only server specific permissions will apply for users on this server
include-global=false

# If users on this server should have their global world permissions applied.
# If set to false, only world specific permissions will apply for users on this server
include-global-world=false

# If users on this server should have global (non-server specific) groups applied
apply-global-groups=true

# If users on this server should have global (non-world specific) groups applied
apply-global-world-groups=true
```

**Comment by lucko** [comment]
Fixed in 82e759d70878b5c7f760feaeef9d1de5b3e84b09
